apiservers/portlayer/restapi/handlers/scopes_handlers.go

```diff
@@ -30,7 +30,12 @@ import (
 
 // ScopesHandlersImpl is the receiver for all of the storage handler methods
 type ScopesHandlersImpl struct {
-	ctx *network.Context
+}
+
+var ctx *network.Context
+
+func GetNetworkContext() *network.Context {
```

Comment:
this puts this ctx into package level and with a very confusing name. ctx used for context.Context extensively in vic repo. 

Why are we putting this variable into package level scope? If we really need to do that can we at least rename it to something else.